### ISSUE TITLE
Weapon modifications OP

### ISSUE THREAD

**Issue by willwolfram18** [stevemer/183siege#4]
After picking up a new weapon (especially the bow), enemies become exceptionally trivial.  We can fix this by upping enemy health values, like we did to the damage.

**Comment by stevemer** [comment]
scaling equivalently to what? Weapon strength?

**Comment by willwolfram18** [comment]
Yeah, exactly.  Many of the enemies have 1-3 health when I encounter them, meaning it's essentially one shot on any enemy once I pick up a new weapon since the minimum damage of the weapons is 5.

**Comment by stevemer** [comment]
Proposed fix?
